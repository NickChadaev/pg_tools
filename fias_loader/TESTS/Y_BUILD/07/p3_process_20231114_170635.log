2023-11-14 17:06:35 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_07 postgres /home/n.chadaev@abrr.local/Y_BUILD/07/stage_gar_c_07.csv 2023-11-13
--------------------------------------------------------
[17:06:35] ... Кабардино-Балкарская Респ, id_region: 8, Execute: /home/n.chadaev@abrr.local/Y_BUILD/07/stage_gar_c_07.csv, process_id:31561
[17:06:35] ... Start process
[17:06:35] ...  -- Version
[17:06:35] ...  -- Очистка от данных
[17:06:35] ...  -- Убираем индексы
[17:06:35] ...  -- Set UNLOGGED
[17:06:35] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[17:06:35] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_REESTR_OBJECTS_20231113_1efa9600-c3c7-4b86-9167-7d7318eb2010.XML
[17:10:13] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_ADDR_OBJ_20231113_80620244-4157-4ae5-acbf-1de827ccb3f2.XML
[17:10:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_HOUSES_20231113_3f1f8948-63bf-450a-b0f2-445eeb697c25.XML
[17:13:30] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_STEADS_20231113_b5b87846-750d-4181-a15b-c2c5ee055722.XML
[17:13:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_APARTMENTS_20231113_518404d7-a8c4-472c-b573-585f96ef3261.XML
[17:15:02] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_ROOMS_20231113_9365bfe0-6dd3-49d2-9937-2dad3939b39a.XML
[17:15:02] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_CARPLACES_20231113_01e18cf0-5ee5-406b-86e1-8bd0635a06f4.XML
[17:15:02] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_ADM_HIERARCHY_20231113_3083b344-46d3-4c71-a5d2-5c2696d778d7.XML
[17:20:03] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_MUN_HIERARCHY_20231113_dffe8e33-af2c-4288-8ec2-97c5dea11937.XML
[17:24:17] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_CHANGE_HISTORY_20231113_2a37e6a2-c624-4bbb-8dfb-81636b6efadb.XML
[17:36:55] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_ADDR_OBJ_DIVISION_20231113_fe767096-566d-46f1-a8f7-45375e12be86.XML
[17:36:55] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_ADDR_OBJ_PARAMS_20231113_ad51912e-9f05-4ed2-a220-52d3b6826544.XML
[17:37:37] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_STEADS_PARAMS_20231113_6f19c5f8-f518-47f1-a0ee-1d63ff26ee74.XML
[17:38:51] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_HOUSES_PARAMS_20231113_6a654bf0-4fdf-4567-99d0-aa04dc7b12ee.XML
[18:10:46] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_APARTMENTS_PARAMS_20231113_7ca2e21d-4a08-4624-a8bd-ffd3f927fe6b.XML
[18:13:09] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_ROOMS_PARAMS_20231113_572c9991-8518-437b-82c8-ffb37a4ab916.XML
[18:13:09] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_CARPLACES_PARAMS_20231113_d6b050c0-06e2-4d17-b224-29baebf6c59e.XML
[18:13:10] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/07/AS_NORMATIVE_DOCS_20231113_24ae0501-6e91-4847-ba98-d429d550305e.XML
[18:13:25] ...  -- Data finish
[18:13:25] ...  -- Set LOGGED
[18:13:34] ... Stop process
--------------------------------------------------------
