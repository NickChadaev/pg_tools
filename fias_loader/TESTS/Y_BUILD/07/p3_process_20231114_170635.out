 save_gar_version 
------------------
               85
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                85
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 306928
gar_fias.as_addr_obj: 5259
gar_fias.as_houses: 232998
gar_fias.as_steads: 8150
gar_fias.as_apartments: 110506
gar_fias.as_rooms: 87
gar_fias.as_carplaces: 0
gar_fias.as_adm_hierarchy: 384150
gar_fias.as_mun_hierarchy: 331172
gar_fias.as_change_history: 1055573
gar_fias.as_addr_obj_division: 12
gar_fias.as_addr_obj_params: 58364
gar_fias.as_steads_params: 101999
gar_fias.as_houses_params: 2524384
gar_fias.as_apartments_params: 192273
gar_fias.as_rooms_params: 97
gar_fias.as_carplaces_params: 0
gar_fias.as_normative_docs: 20414
